assets/css/solcast.webflow.style.css

```diff
@@ -1,23 +1,23 @@
-@import url(./splide-core.min.css);
+@import url(https://solcast.github.io/static/assets/css/splide-core.min.css);
```

Comment:
@Rework-Digital this will 100% stop working after we update the CSP header policy - relative URLs will work because they come from the same domain

Reply:
Won't relative links also fail as they will be looking in solcast.com or solcast.io domains rather then the github repo?